webcomponents/tool/src/main/frontend/packages/sakai-grades/src/SakaiGrades.js

```diff
@@ -7,7 +7,10 @@ import { ASSIGNMENT_A_TO_Z, ASSIGNMENT_Z_TO_A, COURSE_A_TO_Z
 
 export class SakaiGrades extends SakaiPageableElement {
 
-  static properties = { _i18n: { state: true } };
+  static properties = {
+    _i18n: { state: true },
+    secret: { type: Boolean },
```

Comment:
```suggestion
    secret: { type: Boolean },
    _i18n: { state: true },
```

Reply:
I always put the actual attributes first, and the state (private) type stuff afterwards.